lib/inputstreamhelper/__init__.py

```diff
@@ -546,6 +546,61 @@ def _chromeos_config(self):
 
         return devices
 
+    def _backup_libwv(self):
+        """Creates backup of installed lib, keeping multiple backups according to setting 'backups'"""
+        backups = int(get_setting('backups', '2'))
```

Comment:
I would default to 4 or 5. Since we are talking about the library itself, the disk space required is quite limited.

Reply:
ok

Reply:
changed it to 4.